lib/cache/sfCache.class.php

```diff
@@ -50,7 +50,7 @@ public function __construct($options = array())
      *
      * * lifetime (optional): The default life time (default value: 86400)
      *
-     * @throws <b>sfInitializationException</b> If an error occurs while initializing this sfCache instance
+     * @throws sfInitializationException</b> If an error occurs while initializing this sfCache instance
```

Comment:
The closing tag should also be removed? I didn't create a suggestion, as it has to be removed in all changed files.

Reply:
@connorhu please remove the `</b>` tag too :+1: 

Did you check if we have HTML tags elsewhere in the docblocks?

Reply:
What the hell. How did I miss them?